src/CMakeLists.txt

```diff
@@ -604,6 +604,8 @@ add_library(Halide ${HALIDE_LIBRARY_TYPE}
   ${INITIAL_MODULES}
 )
 
+target_compile_options(Halide PUBLIC $<$<CXX_COMPILER_ID:MSVC>:/MP>)
```

Comment:
It's worth adding a comment here that basically replicates the PR comment, otherwise LGTM